bitrise.yml

```diff
@@ -7,12 +7,46 @@ trigger_map:
     workflow: app-store-build
   - pull_request_target_branch: develop
     workflow: qa-build
+  - pull_request_target_branch: develop
+    workflow: test
 workflows:
-  app-store-build:
+  #Reusable workflow that is ran before each workflow to setup the necessary information
+  _setup:
```

Comment:
Creating a re-usable setup step for the workflows that can run before each workflow.